sphinxcontrib/openapi.py

```diff
@@ -122,10 +122,27 @@ def _httpresource(endpoint, method, properties):
     yield ''
 
 
-def openapi2httpdomain(spec):
+def _clean_paths_param(paths):
+    """ Clean paths string param
+
+    - Split string on commas to get list of paths to filter by
+    - Return None if unable to parse the string or paths not provided
+
+    """
+    if paths is None:
+        return None
+    paths = [p.strip() for p in str(paths).split(',')]
+    return paths if len(paths) > 0 else None
+
+
+def openapi2httpdomain(spec, paths=None):
```

Comment:
Hm.. It seems like in time there might be more options (I even can imagine using the `encoding` option while resolving JSON references). If so, it might be better to receive `**options` and do extractions inside rather than extending a function interface. I think it might even simplify a bit `paths` retrieval since we will have an opportunity to use `.get(...)` approach. E.g.:

```python
def openapi2httpdomain(spec, **options):
   endpoints = options.get('endpoints', spec['paths'])
```

Opinion?